### PR TITLE
Per Sensor temp_ignore_limits

### PR DESCRIPTION
**Pull request by Zeanon**
modifies temp_ignore_limits to take a list of sensors that should be ignored instead of ignoreing all

- [x] pr title makes sense
- [x] squashed to 1 commit
- [x] added a test case if possible
- [ ] if new feature, added to the readme
- [ ] ci is happy and green

### CODE REVIEW

**Comment by Zeanon** [comment]
I ran black on the files but it doesnt want to modify the spi_temperature but the ci doesnt like it, I dont really know where the issue is

File: docs/Config_Reference.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ A collection of DangerKlipper-specific system options
 #   Tolerance (in mm) for distance moved in the second homing. Ensures the
 #   second homing distance closely matches the `min_home_dist` when using
 #   sensorless homing. The default is 0.5mm.
-#temp_ignore_limits: False
+#temp_ignore_limits:
 #   When set to true, this parameter ignores the min_value and max_value
```
Comment (rogerlz) [inline docs/Config_Reference.md:98]: missed updating the docs

**Comment by rogerlz** [comment]
> I ran black on the files but it doesnt want to modify the spi_temperature but the ci doesnt like it, I dont really know where the issue is

sorted

**Comment by rogerlz** [comment]
I think we should also support `temp_ignore_limits: all`. I also feel that `self.name in get_danger_options().temp_ignore_limits` could be DangerOptions() method.

**Comment by rogerlz** [comment]
as we talked over discord, lets take the opportunity to make it per sensor.. the danger-option would stay as is to disable all at once.
feel free to close this pr and start a new one with the new implementation if you feel like
ty

**Review by rogerlz — CHANGES_REQUESTED** [review]
as we talked over discord, lets take the opportunity to make it per sensor.. the danger-option would stay as is to disable all at once.
feel free to close this pr and start a new one with the new implementation if you feel like
ty

**Pull request closed without merging**